addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/timing/TimingUtils.java

```diff
@@ -55,73 +61,92 @@ public class TimingUtils {
      */
     public static boolean checkTimingDependence(
             int requestsLimit,
-            double secondsLimit,
+            int highSleepTimeSeconds,
```

Comment:
Document this parameter `highSleepTimeSeconds`.

Reply:
Documented